sklearn/tree/_criterion.pyx

```diff
@@ -667,20 +714,34 @@ cdef class RegressionCriterion(Criterion):
         self.weighted_n_samples = weighted_n_samples
         self.weighted_n_node_samples = 0.
 
+        self.sq_sum_total = 0.0
```

Comment:
This is defined in `pointer_reset` already.

Reply:
Thank you!